shared/actions/profile/index.js

```diff
@@ -209,7 +217,7 @@ function* _profileSaga(): Saga.SagaGenerator<any, any> {
   )
   yield Saga.safeTakeEveryPure(ProfileGen.outputInstructionsActionLink, _outputInstructionsActionLink)
   yield Saga.safeTakeEveryPure(ProfileGen.showUserProfile, _showUserProfile)
-  yield Saga.safeTakeEveryPure(AppGen.link, _onAppLink)
```

Comment:
this can still be pure, the second param is state so just use that and not the select

Reply:
Ah, right, fixed.